modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ItemActionForm.tsx

```diff
@@ -44,32 +74,38 @@ const MESSAGE_TYPES = [
 	},
 ];
 
-const TYPES = [
+const MODAL_VARIANTS = [
```

Comment:
Not sure about the content of this dropdown based on the [Research document](https://liferay.atlassian.net/wiki/spaces/ENGFRONTENDINFRA/pages/2418148860/LPS-186947+-+Tech+research+for+actions+in+dataset+views#Types-of-actions)

> “Full Screen Modal“, “Small Modal“, “Large Modal“. Opens a modal in respective size. We should consider :fire: these types, replacing them with size attribute.

Reply:
Both FDS modal and `openModal` util use `ClayModal` under the hood:
- [FDS modal](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/modal/Modal.js#L138)
- [openModal](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/Modal.js#L203)

(We plan to migrate FDS modal to openModal)

ClayModal supports [these sizes](https://github.com/liferay/clay/blob/master/packages/clay-modal/src/types.ts#L8), so your list is correct.

I'd only make two small changes:
1. Name can be `MODAL_SIZES`, to be more descriptive.
2. Label for `lg` should be `large`, label for `sm` should be `small`.